business/services.go

```diff
@@ -47,7 +47,15 @@ func (in *SvcService) GetServiceList(namespace string) (*models.ServiceList, err
 	go func() {
 		defer wg.Done()
 		var err2 error
-		svcs, err2 = in.k8s.GetServices(namespace, nil)
+		// Check if namespace is cached
+		if kialiCache != nil && kialiCache.CheckNamespace(namespace) {
+			// Cache uses Kiali ServiceAccount, check if user can access to the namespace
+			if _, err := in.businessLayer.Namespace.GetNamespace(namespace); err == nil {
```

Comment:
Here again err is not reported to channel

Reply:
Yeah, make lint catched some but not all.
I will look at them, thanks !